entities/auth.entities.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+
+const { Schema, model } = mongoose;
+
+const AuthSchema = new Schema({
+  slackId: { type: String },
+  slackData: {
+    team: {
+      id: { type: String },
+      name: { type: String },
+    },
+    enterprise: { type: String },
+    user: {
+      token: { type: String },
+      scopes: { type: String },
+      id: { type: String },
+    },
+    tokenType: { type: String },
+    isEnterpriseInstall: { type: Boolean },
+    appId: { type: String },
+    authVersion: { type: String },
+    bot: {
+      scopes: [String],
+      token: { type: String },
+      userId: { type: String },
+      id: { type: String },
+    },
+    incomingWebhook: {
+      url: { type: String },
+      channel: { type: String },
+      channelId: { type: String },
```

Comment:
what's the difference between `channel` and `channelId`?

Reply:
the channel is the name of the channel and channeled is the id.